python/ray/dag/compiled_dag_node.py

```diff
@@ -765,7 +765,18 @@ def _preprocess(self) -> None:
         self.input_task_idx, self.output_task_idx = None, None
         self.actor_task_count.clear()
 
-        nccl_actors_p2p: Set["ray.actor.ActorHandle"] = set()
+        # Actors and DAG nodes involved in NCCL P2P send/recv with `transport=nccl`,
+        # in which case the default NCCL group is used. The default NCCL group is the
+        # `custom_nccl_group` in `experimental_compile` if specified. If not, we
```

Comment:
Update comments to explain how `default` is decided.
1. If a `custom` is specified ...
2. Reuse ...
3. Create ...

Reply:
Done.